packages/superset-ui-control-utils/src/InfoTooltipWithTrigger.tsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { kebabCase } from 'lodash';
+// @ts-ignore
+import { Tooltip, OverlayTrigger } from 'react-bootstrap';
```

Comment:
Prefer to remove `// @ts-ignore` and add proper typing support by installing `@types/react-bootstrap`.

Reply:
If I remember correctly, bootstrap 3 doesn't have types support. We ran into this problem in incubator-superset as well.

Reply:
Instead of @ts-ignore, you can add a declaration file. Even if it is an empty declaration

Example: 
https://github.com/apache-superset/superset-ui/blob/master/packages/superset-ui-translation/types/external.d.ts

Reply:
I think `@types/react-bootstrap @ 0.32.2` works with Bootstrap v3. There might be cases with suboptimal typings, where you can then do `@ts-ignore` per module.

Reply:
Nice! That means we can add types for bootstrap back in incubator-superset too

Reply:
@ktmud I tried adding the types for bootstrap, but that leads to an issue in CI. It looks like our version of typescript isn't happy with those type definitions, any ideas on how to fix?

Reply:
Sorry, should've used `@types/react-bootstrap @ ^0.32.21`. This is also what's used in `incubator-superset`.

Btw, it's preferable to have `@types/*` in `dependencies` instead of `devDependencies` so that dependent packages can have easier access to typings when the library is used in `npm link`.